components/HeroSection.jsx

```diff
@@ -13,17 +14,29 @@ export const HeroSection = ({
   description,
   PrimaryButton,
   SecondaryButton,
+  isUnderConstruction,
```

Comment:
it's now specified to business logic, we could call it something like "statusTag?: React.ReactNode"
and render as is if passed. with this approach we could possibly use this component in pages/agents/shorts.jsx